test/WebJobs.Script.Tests/NodeEndToEndTests.cs

```diff
@@ -187,24 +187,6 @@ public void ErrorFunction_DoesNotBringDownHost()
         }
 
         [Fact]
-        public async Task ManualTrigger()
```

Comment:
Pushed the stuff this guy was testing related to logging into its own scenario test